pkg/cortex/modules.go

```diff
@@ -170,6 +171,23 @@ func (t *Cortex) initOverrides() (serv services.Service, err error) {
 	return nil, err
 }
 
+func (t *Cortex) initOverridesExporter() (services.Service, error) {
+	cfgChan := tenantLimitsRuntimeConfigChannel(t.RuntimeConfig)
+	if cfgChan == nil {
```

Comment:
```suggestion
	if t.Cfg.isModuleEnabled(OverridesExporter) && cfgChan == nil {
```

This checks if overrides exporter has been explicitly enabled.